example/lib/screens/message_view.dart

```diff
@@ -4,9 +4,8 @@ import 'package:airship_example/styles.dart';
 
 class MessageView extends StatelessWidget {
   final InboxMessage message;
-  final updateParent;
 
-  MessageView({this.message, this.updateParent});
```

Comment:
whats up with all these updateParent things?

Reply:
(this is just for my better understanding, not saying I think the change is wrong)

Reply:
updateParent was basically a completion handler that would allow the message to make a function call in the parent context. The update state function it was calling was redundant - as the parent will update its state as it comes back into view.